pkg/mqtrigger/mqtrigger.go

```diff
@@ -40,10 +44,68 @@ func Start(logger *zap.Logger, routerUrl string) error {
 	// Message queue type: nats is the only supported one for now
 	mqType := os.Getenv("MESSAGE_QUEUE_TYPE")
 	mqUrl := os.Getenv("MESSAGE_QUEUE_URL")
+
+	// For authentication with message queue
+	secrets, err := readSecrets(logger)
+
+	if err != nil {
+		panic(err)
+	}
+
 	mqCfg := messageQueue.MessageQueueConfig{
-		MQType: mqType,
-		Url:    mqUrl,
+		MQType:  mqType,
+		Url:     mqUrl,
+		Secrets: secrets,
 	}
 	messageQueue.MakeMessageQueueTriggerManager(logger, fissionClient, routerUrl, mqCfg)
 	return nil
 }
+
+func getCurrentNamespace() string {
```

Comment:
I think you can remove this function since we don't need it anymore.

Reply:
I will remove this. Thanks for pointing this out :)

Reply:
Done